pwa/src/components/topNav/TopNav.css

```diff
@@ -0,0 +1,29 @@
+.TopNav {
+  border-bottom: var(--utrecht-space-inline-3xs) var(--utrecht-color-red-40)
+    solid;
+}
+
+.TopNav .TopNav-inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  padding-block-start: var(--utrecht-space-inline-2xs);
+  padding-block-end: var(--utrecht-space-inline-2xs);
+}
+
+.TopNav .TopNav-inner > svg {
+  max-block-size: 100px;
```

Comment:
I think we should make our own design token for this value.

Token could be something along the lines of `--conduction-nav-logo-height-lg`.

Location of these tokens would be `pwa/src/styling/design-tokens/conduction-design-tokens.css`